### PR TITLE
Add org.gnome.GScope.json

### PR DESCRIPTION
**Pull request by carlocaione**
gScope is source browsing tool for large and complex projects written in C.

Signed-off-by: Carlo Caione <ccaione@baylibre.com>

### CODE REVIEW

File: org.gnome.GScope.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "gtksourceview",
+            "config-opts" : [ "--disable-Werror" ],
```
Comment (bilelmoussaoui) [inline org.gnome.GScope.json]: You could disable docs here :)

File: org.gnome.GScope.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
```
Comment (bilelmoussaoui) [inline org.gnome.GScope.json]: Those cflags & cxxflags are not needed anymore on GNOME Runtime 3.30

**Comment by bilelmoussaoui** [comment]
Btw, I don't think that you have the rights to use org.gnome.GScope as an app-id.

File: org.gnome.GScope.json (hunk 1 above)
Comment (carlocaione) [reply org.gnome.GScope.json]: yup

File: org.gnome.GScope.json (hunk 2 above)
Comment (carlocaione) [reply org.gnome.GScope.json]: ok

**Comment by carlocaione** [comment]
> Btw, I don't think that you have the rights to use org.gnome.GScope as an app-id.

Why is that?

**Comment by bilelmoussaoui** [comment]
Because you don't own that domain name and it's not part of GNOME Core apps. Nor it's on gitlab.gnome.org/World

**Comment by bilelmoussaoui** [comment]
Looking at the appdata file, https://gitlab.gnome.org/carlo.caione/gscope/blob/master/data/org.gnome.GScope.appdata.xml.in
It lacks some stuff:
- The app-id is incorrect as I said before, plus you don't need the `.desktop`
- You need the `launchable` tag
- You could add `provides` tag
- `releases` tag is missing
- OARS tags are missing
- Screenshots are missing in the appdata file
- You could add issues & donate, translation urls if you have them :)
- The project group is incorrect. You could replace that with developer_name 
- You should also add translation to your appdata file to show up if the app is translated on the user's language 
For the desktop file https://gitlab.gnome.org/carlo.caione/gscope/blob/master/data/org.gnome.GScope.desktop.in
You could also add those two to your desktop file GenericName,Comment

**Comment by nedrichards** [comment]
This thread explains the position on org.gnome app-ids. https://github.com/flathub/flathub/pull/726#discussion_r234535571 it's totally possible to get approved, just pop an email to GNOME.

**Comment by carlocaione** [comment]
Closing this PR. Opening a new one with a new id.

**Pull request closed without merging**